### ISSUE TITLE
still the problem with equality constraints

### ISSUE THREAD

**Issue by xiaoyuanzh** [liuq/QuadProgpp#13]
If I put CE and ce0 to 0
it says
terminate called after throwing an instance of 'std::runtime_error'
  what():  Constraints are linearly dependent

**Comment by liuq** [comment]
Yes, this is true since the equality constraints must be linearly independent. If you do not have equality constraints set their number to zero.
